src/python/WMCore/Storage/RucioFileCatalog.py

```diff
@@ -239,9 +242,61 @@ def rseName(currentSite, currentSubsite, storageSite, volume):
     except Exception as ex:
         msg = "RucioFileCatalog.py:rseName() Error reading storage.json: %s\n" % storageJsonName
         msg += str(ex)
-        raise RuntimeError(msg)
+        raise RuntimeError(msg) from ex
     for jsElement in jsElements:
         if jsElement['site'] == storageSite and jsElement['volume'] == volume:
             rse = jsElement['rse']
             break
     return rse
+
+def get_default_cmd(currentSite, currentSubsite, storageSite, volume, protocolName):
+    """
+    Return default command for a protocol, for example:
+    https://gitlab.cern.ch/SITECONF/T1_DE_KIT/-/blob/master/storage.json?ref_type=heads#L17
+    :currentSite is the site where jobs are executing
+    :currentSubsite is the sub site if jobs are running here
+    :storageSite is the site for storage
+    :volume is the volume name, for example:
+        https://gitlab.cern.ch/SITECONF/T1_DE_KIT/-/blob/master/storage.json?ref_type=heads#L3
+    :protocolName is the 'protocol' in site-local-config.xml under stageOut
+    """
+
+    storageJsonName = storageJsonPath(currentSite, currentSubsite, storageSite)
+    try:
+        with open(storageJsonName, encoding="utf-8") as jsonFile:
+            jsElements = json.load(jsonFile)
+    except Exception as ex:
+        msg = "RucioFileCatalog.py:getDefaultCmd() Error reading storage.json: %s\n" % storageJsonName
```

Comment:
Can you please update this error message to keep consistent with the actual method name?
Or maybe I would suggest to rephrase it to something like (the module and line number are already present in the logs):
```
        msg = f"Failed to open storage.json: {storageJsonName}\n. Error: {str(ex)}"
```